tools/gulp/packages.ts

```diff
@@ -3,8 +3,12 @@ import {join} from 'path';
 
 export const cdkPackage = new BuildPackage('cdk');
 export const materialPackage = new BuildPackage('material', [cdkPackage]);
-export const examplesPackage = new BuildPackage('material-examples', [materialPackage, cdkPackage]);
 export const momentAdapterPackage = new BuildPackage('material-moment-adapter', [materialPackage]);
+export const examplesPackage = new BuildPackage('material-examples', [
+  materialPackage,
+  cdkPackage,
+  momentAdapterPackage
```

Comment:
If you only add `momentAdapterPackage`, then it won't build `material` or the `cdk` a second time. The dependencies of the `momentAdapterPackage` will be installed automatically (see other packages)

Reply:
Done.